aspnet/web-forms/overview/getting-started/getting-started-with-aspnet-45-web-forms/display_data_items_and_details.md

```diff
@@ -145,7 +163,7 @@ Now you can run the application to see an individual product displayed based on
 
 ## Summary
 
-In this tutorial of the series you have add markup and code to display a product list and to display product details. During this process you have learned about strongly typed data controls, model binding, and value providers. In the next tutorial, you'll add a shopping cart to the Wingtip Toys sample application.
+In this series tutorial, you added markup and code to display products and  product details. During this process you've learned about strongly typed data controls, model binding, and value providers. In the next tutorial, you'll add a shopping cart to the Wingtip Toys sample application.
```

Comment:
* Move this "Summary" section to end and rename it Next steps
* "series tutorial" --> tutorial
* End with the next/prev links that are currently there.